File: clients/asset-client/src/constants/TypeOrganizationConstant.ts
import { createConstant } from "./Constant";

const TypeOrganizationConstant = createConstant(
  {
    Company: "Company",
    Personal: "Personal",
    Organization: "Organization",
  } as const,
  {
    Company: { displayName: "Doanh nghiệp" },
    Personal: { displayName: "Cá nhân" },
    Organization: { displayName: "Tổ chức" },
  }
);

export default TypeOrganizationConstant;

File: services/kpi-service/Hinet.Service/Constant/TypeOrganizationConstant.cs
using System.ComponentModel;

namespace Hinet.Service.Constant
{
    public class TypeOrganizationConstant
    {
        [DisplayName("Doanh nghiệp")]
        public static string Company => "Company";

        [DisplayName("Cá nhân")]
        public static string Personal => "Personal";

        [DisplayName("Tổ chức")]
        public static string Organization => "Organization";
    }
}
